src/components/CohortBuilder/Categories/index.js

```diff
@@ -95,7 +97,7 @@ class Categories extends React.Component {
           color={theme.studyRed}
           anchorId={'anchor-study'}
         >
-          <StudyIcon fill={theme.studyRed} />
+          <StudyIcon fill={theme.studyRed} width="14px" height="17px" />
```

Comment:
is it the issue Adrian was talking about to pass fill and classname ?

Reply:
☝️

Reply:
yes

Reply:
the issue was that no `fill` prop was passed down to the icon. We (me and Adrian) added the prop. The `width="14px" height="17px" ` (similarly to the rest of the icons in the file) was added so that the icon be the same size as the other icons in this component (pls, see the whole file).

Reply:
the icons in the portal are a complete mess. they are all different, and don't work in the same way. Anyway, its such a tedious job that nobody wants to tackle it. :)

Reply:
if there is no way to fix it, i'm confortable to let it like this

Reply:
But the icon is in `src/icons`, why not 'normalize' our icons there instead of a per use basis.

Reply:
@adipaul1981 🐥 🥚